### ISSUE TITLE
crash in DPI calculation "decimal.InvalidOperation: quantize result has too many digits"

### ISSUE THREAD

**Issue by azrdev** [jbarlow83/OCRmyPDF#114]
Using the PDF of [this paper](http://ieeexplore.ieee.org/abstract/document/7417297/), I get the following backtrace. ocrmypdf version is 4.3.2 on archlinux.
~~~
  ERROR - Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/ruffus/task.py", line 751, in run_pooled_job_without_exceptions
    register_cleanup, touch_files_only)
  File "/usr/lib/python3.5/site-packages/ruffus/task.py", line 567, in job_wrapper_io_files
    ret_val = user_defined_work_func(*params)
  File "/usr/lib/python3.5/site-packages/ocrmypdf/__main__.py", line 584, in repair_pdf
    pdfinfo.extend(pdf_get_all_pageinfo(output_file))
  File "/usr/lib/python3.5/site-packages/ocrmypdf/pageinfo.py", line 408, in pdf_get_all_pageinfo
    return [_pdf_get_pageinfo(infile, n) for n in range(pdf.numPages)]
  File "/usr/lib/python3.5/site-packages/ocrmypdf/pageinfo.py", line 408, in <listcomp>
    return [_pdf_get_pageinfo(infile, n) for n in range(pdf.numPages)]
  File "/usr/lib/python3.5/site-packages/ocrmypdf/pageinfo.py", line 391, in _pdf_get_pageinfo
    page, pageinfo, contentsinfo)]
  File "/usr/lib/python3.5/site-packages/ocrmypdf/pageinfo.py", line 390, in <listcomp>
    pageinfo['images'] = [im for im in _find_page_images(
  File "/usr/lib/python3.5/site-packages/ocrmypdf/pageinfo.py", line 339, in _find_page_images
    yield from _find_page_regular_images(page, pageinfo, contentsinfo)
  File "/usr/lib/python3.5/site-packages/ocrmypdf/pageinfo.py", line 330, in _find_page_regular_images
    image['dpi_w'] = Decimal(image['dpi_w']).quantize(DPI_PREC)
  File "/usr/lib/python3.5/_pydecimal.py", line 2562, in quantize
    'quantize result has too many digits for current context')
  File "/usr/lib/python3.5/_pydecimal.py", line 4032, in _raise_error
    raise error(explanation)
decimal.InvalidOperation: quantize result has too many digits for current context
~~~

**Comment by jbarlow83** [comment]
The paper is behind a paywall. Can you send me a link to the file some other way?

**Comment by jbarlow83** [comment]
Never mind, I found another PDF that produces this behavior. Simple problem, easy fix. You can delete the line "decimal.getprec() = 6" from ocrmypdf/pageinfo.py if you're in a hurry, that seems to be all that's needed.

**Comment by jbarlow83** [comment]
Fixed 4.3.4

**Comment by azrdev** [comment]
@jbarlow83 thanks!
